packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/meta/DependencyInjection.kt

```diff
@@ -135,3 +135,15 @@ fun <E : Any, T : Any> SharedFlow<E>.latest(
         onChange,
         defaultValue
     )
+
+fun <E : Any> SharedFlow<E>.eventCount(
```

Comment:
This is a convenient method to have an event counter that will become the modification count of the components in the future whenever we implement the ModificationTracker interface.